conference/DS/ics.yml

```diff
@@ -0,0 +1,16 @@
+- title: ICS
+  description: ACM International Conference on Supercomputing
+  sub: DS
+  rank: B
+  dblp: ics
+  confs:
+    - year: 2024
+      id: ics24
+      link: https://ics2024.github.io/
+      timeline:
+        - abstract_deadline: '2024-01-12 19:59:59'
+          deadline: '2024-01-19 19:59:59'
+          comment: 'Registration deadline: Friday Jan 12, 2024, 11:59:59 AM UTC'
+      timezone: UTC+8
```

Comment:
should be AoE？

Reply:
You mean only AoE is supported here? I think UTC+8 is correct. From the hotcrp, 19:59 is our time(UTC+8), while the AoE is one day earlier (2024-01-11 and 2024-01-18).

Reply:
> You mean only AoE is supported here? I think UTC+8 is correct. From the hotcrp, 19:59 is our time(UTC+8), while the AoE is one day earlier (2024-01-11 and 2024-01-18).

This project will automatically convert to Beijing time based on "timezone" and does not require manual conversion.

Reply:
The official deadline timezone for ICS 2024 is AoE, and this project will adjust the time based on the user's local timezone (may be UTC+8, +7 or any others), as explained by @Lukangkang123.
<img width="479" alt="image" src="https://github.com/ccfddl/ccf-deadlines/assets/20631913/a8ccacf6-0976-4fcf-9127-12b1a9223b2c">

Reply:
> The official deadline timezone for ICS 2024 is AoE, and this project will adjust the time based on the user's local timezone (may be UTC+8, +7 or any others), as explained by @Lukangkang123. Therefore, the correct item may be sth. like [23:59:59 AoE]. <img alt="image" width="479" src="https://private-user-images.githubusercontent.com/20631913/293815848-a8ccacf6-0976-4fcf-9127-12b1a9223b2c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQyNDcwOTAsIm5iZiI6MTcwNDI0Njc5MCwicGF0aCI6Ii8yMDYzMTkxMy8yOTM4MTU4NDgtYThjY2FjZjYtMDk3Ni00ZmNmLTkxMjctMTJiMWE5MjIzYjJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTAzVDAxNTMxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk3MDNlODJhZDc4N2QwYTE4Y2RmYTU0N2RhYjM0YTBlNjA1YjRhYTJkMDA5NWE2NjQyZmFkY2I2NGFiN2E0NDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Ac3jrDBYtgroAjQtna935ubu4AlKlRFrCasRaSsye6A">

OK，I will update it